platform/osgi/platform-osgi-configadmin/src/test/java/org/codice/felix/cm/file/EncryptingPersistenceManagerTest.java

```diff
@@ -64,16 +66,28 @@
 
   @Mock private PersistenceManager mockManager;
 
-  private EncryptingPersistenceManager persistenceManager;
-
   @Before
   public void setup() throws Exception {
-    String pwDirectory = temporaryFolder.newFolder().getAbsolutePath();
-    persistenceManager = new EncryptingPersistenceManager(mockManager, pwDirectory);
+    temporaryFolder.create();
```

Comment:
❓ Why are you managing your own temp folder creation & deletion?
https://junit.org/junit4/javadoc/4.12/org/junit/rules/TemporaryFolder.html